common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_monitor.yaml

```diff
@@ -159,14 +159,6 @@ cACTION_MONITOR_CLIENT_STATISTICS_11K_RESPONSE:
   _type: class
   params: sStatisticsResponse11k
 
```

Comment:
this commit and the previous one (of the auto-generated code) should be in the same commit IMO

Reply:
accepted